lib/node_modules/@stdlib/random/base/README.md

```diff
@@ -156,10 +156,86 @@ r = rand();
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var objectKeys = require( '@stdlib/utils/keys' );
+
 var random = require( '@stdlib/random/base' );
+var arcsine = random.arcsine;
```

Comment:
This isn't what we are looking for. This essentially just takes various examples from `@stdlib/random/base/arcsine` and puts them in a single script. That isn't particularly interesting, and the code as written below does not conform to project conventions. As such, this PR will need significant changes before being ready for merge.